gatsby-config.js

```diff
@@ -93,15 +93,19 @@ module.exports = {
               },
               {
                 title: "AEM 6.3 and prior Cloud Services configuration",
-                path: "guides/using/aem/aem_cloud_service_config.md",
+                path: "guides/using/aem/aem_cloud_service_config_6.3.md",
               },
               {
                 title: "AEM - Adobe I/O Events - advanced configurations",
                 path: "guides/using/aem/aem_advanced_configurations.md",
               },
               {
-                title: "Adobe I/O Events Sling Health Checks",
-                path: "guides/using/aem/aem_on_premise_healthcheck.md",
+                title: "Adobe I/O Events (AEM 6.4 and prior) Sling Health Checks",
+                path: "guides/using/aem/aem_healthcheck_6.4.md",
```

Comment:
Shall we hide this from the sidebar since we already have the link in the `aem_on_premise_install_6.3` page?

Reply:
yes